docs/notebooks/analysis/swept_langmuir/find_floating_potential.ipynb

```diff
@@ -177,21 +203,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "(results[2], results.rsq)"
+    "(extras[1], extras.rsq)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "- `results[3]` = `results.func` = the resulting fitted function\n",
-    "\n",
-    "    - `results.func` is a callable representation of the fitted function `I = results.func(V)`.\n",
-    "    - `resulst.func` is an instance of a sub-class of `AbstractFitFunction`. ([FitFuction classes](../../../api_static/plasmapy.analysis.fit_functions.rst))\n",
-    "    - Since `results.func` is a class instance, there are many other attributes available.  For example,\n",
-    "        - `results.func.params` is a named tuple of the fitted parameters\n",
-    "        - `results.func.param_errors` is a named tuple of the fitted parameter errors\n",
-    "        - `results.func.root_solve()` finds the roots of the fitted function. This is how $V_f$ is calculated."
+    "[FitFunction]: ../../../api_static/plasmapy.analysis.fit_functions.rst\n",
+    "\n",
+    "- `extras[2]` = `extras.fitted_func` = the resulting fitted function\n",
+    "\n",
+    "    - `extras.fitted_func` is a callable representation of the fitted function `I = extras.fitted_func(V)`.\n",
+    "    - `extras.fitted_func` is an instance of a sub-class of `AbstractFitFunction`. ([FitFunction classes][FitFunction])\n",
+    "    - Since `extras.fitted_func` is a class instance, there are many other attributes available.  For example,\n",
+    "        - `extras.fitted_func.params` is a named tuple of the fitted parameters\n",
+    "        - `extras.fitted_func.param_errors` is a named tuple of the fitted parameter errors\n",
```

Comment:
```suggestion
    "        - `extras.fitted_func.params` is a `namedtuple` of the fitted parameters\n",
    "        - `extras.fitted_func.param_errors` is a `namedtuple` of the fitted parameter errors\n",
```
[optional]  Maybe link to namedtuple in the Python docs?

Reply:
Since this is in a notebook, I'm going to leave it as "named tuple".  Especially since `namedtuple` is a factory function and not a type.  So, saying "`val` is a `namedtuple`" is saying `val` is a factory function for named tuples.

I'm sure there's a better way of wording this, but that's an issue for future me.